src/vs/workbench/contrib/themes/browser/themes.contribution.ts

```diff
@@ -768,7 +768,8 @@ class DefaultThemeUpdatedNotificationContribution implements IWorkbenchContribut
 			];
 			await this._notificationService.prompt(
 				Severity.Info,
-				localize({ key: 'themeUpdatedNotification', comment: ['{0} is the name of the new default theme'] }, "Visual Studio Code now ships with a new default theme '{0}'. If you prefer, you can switch back to the old theme or try one of the many other color themes available.", newTheme.label),
+				// {{SQL CARBON EDIT}} Update application name to Azure Data Studio
+				localize({ key: 'themeUpdatedNotification', comment: ['{0} is the name of the new default theme'] }, "Azure Data Studio now ships with a new default theme '{0}'. If you prefer, you can switch back to the old theme or try one of the many other color themes available.", newTheme.label),
```

Comment:
For any code files under vs you should be creating a new loc entry in https://github.com/Microsoft/azuredatastudio/blob/5c6ea2890ad73dfbb54329b311321e39a7623deb/src/sql/base/common/locConstants.ts and using that. Changes made directly here are overwritten in other languages since our langpacks are partially based off of the VS Code defined strings.

@smartguest FYI